place/views.py

```diff
@@ -1,3 +1,175 @@
+
+from share.decorators import check_auth_decorator
+from share.utils import get_value_from_token
+
+
+class AddPlaceView(View):
+    @transaction.atomic
+    #@check_auth_decorator
+    def post(self, request):
+        try:
+            data       = json.loads(request.body)
+            token      = request.headers['token']
+            user_id    = token #get_value_from_token(token, 'user_id')            
+            categories = Category.objects.filter(name = data['category'])
+        
+            if not categories:
+                return JsonResponse({"message":"NOT_EXIST"}, status=400)
+            
+            category   = categories.get()
+            place      = Place.objects.create(
+                                    address                  = data['address'],
+                                    price_per_hour           = Decimal(data['price_per_hour']),
+                                    area                     = data['area'],
+                                    floor                    = data['floor'],
+                                    maximum_parking_lot      = data['maximum_parking_lot'],
+                                    allowed_members_count    = data['allowed_members_count'],
+                                    description              = data['description'],
+                                    using_rule               = data['using_rule'],
+                                    info_nearby              = data['info_nearby'],
+                                    minimum_rental_hour      = data['minimum_rental_hour'],
+                                    delegate_place_image_url = data['delegate_place_image_url'],
+                                    surcharge_rule           = data['surcharge_rule'],
+                                    category_id              = category.id,
+                                    user_id                  = user_id,
+                                )
+            
+            PlaceImage.objects.bulk_create(
+                [
+                    PlaceImage( 
+                        url      = image['url'],
+                        place_id = place.id
+                    ) for image in data['images']
+                ]
+            )
+
+            for tag in data['tags']:
+                target_tag = None
+                if not Tag.objects.filter(name = tag['tag']).exists():
+                    target_tag = Tag.objects.create(name = tag['tag'])
+                else:
+                    target_tag = Tag.objects.get(name = tag['tag'])
+
+                target_tag.places_tags.add(place)
+
+            InvalidBookingDay.objects.bulk_create(
+                [
+                    InvalidBookingDay(
+                        place_id = place.id,
+                        day      = datetime.strptime(
+                                        day['date'],
+                                        '%Y-%m-%d'
+                                                    )
```

Comment:
```suggestion
                        place_id = place.id,
                        day      = datetime.strptime(day['date'], '%Y-%m-%d')
```
인자가 두가지 뿐이라 한 줄로 합치시는게 가독성이 더 높을 것 같습니다.

Reply:
넵 반영하겠습니다